tomcat/src/main/java/nextstep/jwp/vo/Response.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.jwp.vo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Response {
+
+    private static final String HTTP_1_1 = "HTTP/1.1 ";
+    private static final String BLANK = " ";
+    private static final String DELIMITER = "\r\n";
+    private static final String EMPTY = "";
+    private static final String HEADER_DELIMITER = ": ";
+    private final List<String> headers;
```

Comment:
```suggestion
    private static final String HEADER_DELIMITER = ": ";
    
    private final List<String> headers;
```

static과 local variable은 개행 해주는게 좋을 것 같아요!